cmd/db-server/main.go

```diff
@@ -67,30 +69,40 @@ func createMOServer(inputCtx context.Context, pu *config.ParameterUnit) {
 
 	moServerCtx := context.WithValue(inputCtx, config.ParameterUnitKey, pu)
 	mo = frontend.NewMOServer(moServerCtx, address, pu)
-	{
-		if err := util.SetUUIDNodeID(nil); err != nil {
-			panic(err)
-		}
-		// init trace/log/error framework
-		if _, err := trace.Init(moServerCtx,
-			trace.WithMOVersion(MoVersion),
-			trace.WithNode("node_uuid", trace.NodeTypeNode),
-			trace.EnableTracer(!pu.SV.DisableTrace),
-			trace.WithBatchProcessMode(pu.SV.TraceBatchProcessor),
-			trace.DebugMode(pu.SV.EnableTraceDebug),
-			trace.WithSQLExecutor(func() ie.InternalExecutor {
-				return frontend.NewInternalExecutor(pu)
-			}),
-		); err != nil {
+	var fs fileservice.FileService
```

Comment:
Please use Config.createFileService("local") to get a FileService instance from config file. 
This object contains all services. Use named paths to refer to different services.